server/models/account/user.ts

```diff
@@ -106,12 +123,66 @@ enum ScopeNames {
             }
           }
         ]
-      },
-      {
-        model: UserNotificationSettingModel,
-        required: true
       }
     ]
+  },
+  [ScopeNames.WITH_STATS]: {
+    attributes: {
+      include: [
+        literalVideoQuotaUsed,
+        [
+          literal(
+            '(' +
+              'SELECT COUNT("video"."id") ' +
+              'FROM "video" ' +
+              'INNER JOIN "videoChannel" ON "videoChannel"."id" = "video"."channelId" ' +
+              'INNER JOIN "account" ON "account"."id" = "videoChannel"."accountId" ' +
+              'WHERE "account"."actorId" = "UserModel"."id"' +
```

Comment:
`"account"."userId"`

Reply:
oh, that's why some stats were bogus :sweat: